content/index.id.md

```diff
@@ -99,79 +99,79 @@ Sarana penulisan adalah perangkat lunak dan jasa yang digunakan oleh "para penul
+Agen pengguna termasuk browser, ekstensi browser, pemutar media, perangkat pembaca, dan aplikasi lainnya yang menyajikan konten web.
 
 Informasi seputar UUAG:
 - [Gambaran Umum UAAG](/standards-guidelines/uaag/)
 - [Catatan UAAG 2.0](https://www.w3.org/TR/UAAG20/)
 
-### Draf Kerja Pedoman Aksesibilitas Konten Web (WCAG) 3 W3C {#wcag3}
+### Rancangan Pedoman Aksesibilitas Konten Web (WCAG) 3 W3C {#wcag3}
 
-WCAG 3 adalah draf awal yang ditujukan untuk menjadi Standar W3C. WCAG 3 berlaku untuk konten web, aplikasi, alat, penerbitan, dan teknologi baru dalam web.
+WCAG 3 adalah rancangan awal yang ditujukan untuk menjadi Standar W3C. WCAG 3 berlaku untuk konten web, aplikasi, alat, penerbitan, dan teknologi baru dalam web.
 
 Informasi seputar WCAG 3:
-- [[Pengantar WCAG 3]](/standards-guidelines/wcag/wcag3-intro/)
+- [[Pengenalan ke WCAG 3]](/standards-guidelines/wcag/wcag3-intro/)
 
 ## Spesifikasi Teknis
 
-### Aplikasi Internet Kaya yang Aksesibel (WAI-ARIA) {#aria}
+### Aplikasi Internet yang Kaya dan Aksesibel (WAI-ARIA) {#aria}
 
 ARIA menyediakan semantik agar penulis web dapat menyampaikan perilaku antarmuka pengguna dan informasi struktural ke teknologi pendukung (seperti pembaca layar). Spesifikasi ARIA menyediakan hakikat peran (*roles*), keadaan (*states*), dan properti yang mendefinisikan elemen antarmuka pengguna yang aksesibel.
 
-Rangkaian ARIA mencakup spesifikasi pemetaan <abbr title="Antarmuka Pemrograman Aplikasi">API</abbr> yang menyediakan panduan implementasi agen pengguna. Termasuk juga modul untuk Publikasi Digital dan Grafis.
+Rangkaian ARIA mencakup spesifikasi pemetaan <abbr title="Antarmuka Pemrograman Aplikasi">API</abbr> yang menyediakan panduan implementasi pada agen pengguna. Termasuk juga modul untuk Publikasi Digital dan Grafis.
 
 Informasi seputar ARIA:
-- [Gambaran Umum WAI-ARIA](/standards-guidelines/aria/) – mencakup [daftar dan deskripsi pemetaan modul dan API](/standards-guidelines/aria/#versions)
-- [Praktek Penulisan WAI-ARIA](https://www.w3.org/TR/wai-aria-practices/)
-- [Standar WAI-ARIA 1.1](https://www.w3.org/TR/wai-aria-1.1/)
+- [Gambaran Umum WAI-ARIA](/standards-guidelines/aria/) – mencakup [daftar dan deskripsi dari modul dan pemetaan API](/standards-guidelines/aria/#versions)
+- [Praktik Penulisan WAI-ARIA](https://www.w3.org/TR/wai-aria-practices/)
+- [Standar WAI-ARIA 1.2](https://www.w3.org/TR/wai-aria-1.2/)
 
 ### Audio dan Video {#multimedia}
 
-- [WebVTT: Format Trek Teks Video Web](https://www.w3.org/TR/webvtt/) adalah format untuk teks, deskripsi video berbentuk teks, dan metadata lainnya yang waktunya diselaraskan dengan konten video atau audio.
+- [WebVTT: Format Web Video Text Tracks - Trek Teks pada Video Web](https://www.w3.org/TR/webvtt/) adalah format untuk takarir, deskripsi video berbentuk teks, dan metadata lainnya yang waktunya diselaraskan dengan konten video atau audio.
```

Comment:
@frirasyidi If you think relevant to keep some text in English, I recommend using <span lang="en"> around it.

Reply:
Will be updated in the next commit.